### DISCUSSION TITLE
ScriptReference/Unity.Collections.ReadOnlyAttribute

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#602] [Notes]
# ScriptReference/Unity.Collections.ReadOnlyAttribute

https://docs.unity3d.com/ScriptReference/Unity.Collections.ReadOnlyAttribute.html

**Comment by redwyre** [comment]
By "mark a member of a struct" it seems the docs mean here that it works both directly on containers, or on structs that have containers inside them. For anything that isn't a container (i.e. primitive types, this attribute does nothing.
